### PR TITLE
April month-end release (21.4.0)

### PR DESCRIPTION
**Pull request by mnaik051493**
## Added

* Extended WebSphere Liberty tuning options

### Changed

* Changed the OpenShift reference architecture diagram to provide additional clarity regarding the statefulsets
* Improved pod labels for compatibility with service meshes ([#61](https://github.com/IBM/spm-kubernetes/issues/61))
* Improved navigation links to avoid 404 errors ([#68](https://github.com/IBM/spm-kubernetes/issues/68))

### Removed

* Removed the `wait-for-database` initContainer from producer and consumer pods
  * Validation of the database configuration is already handled by the pre-install hook

### CODE REVIEW

File: helm-charts/mqserver/RELEASENOTES.md [hunk 1]
```diff
@@ -8,6 +8,11 @@
 
 # Version History
 
+## Unreleased
```
Comment (gillesde) [inline helm-charts/mqserver/RELEASENOTES.md:11]: We should bump the version to 1.10.0

Comment (gillesde) [reply helm-charts/mqserver/RELEASENOTES.md:11]: Bump version also in  [helm-charts/mqserver/Chart.yaml at line 32](https://github.com/IBM/spm-kubernetes/pull/71/files#diff-ad8e928f3b55ab4a7764b79885c775600a691dbf884754a553361cd661087a3dR32)

File: helm-charts/spm/Chart.yaml [hunk 2]
```diff
@@ -56,24 +56,24 @@ icon: https://avatars2.githubusercontent.com/u/1459110
...
   - name: uawebapp
-    version: "~3.2.1"
+    version: "~3.2.2"
     repository: "@local-development"
   - name: web
-    version: "~3.2.1"
+    version: "~3.2.2"
     repository: "@local-development"
   - name: mqserver
     version: "~1.9.0"
```
Comment (gillesde) [inline helm-charts/spm/Chart.yaml:71]: Also bump to 1.10.0

File: src/pages/deployment/config-reference.mdx [hunk 3]
```diff
@@ -77,6 +79,12 @@ In the following properties, `<applicationID>` is a placeholder for the name of
...
 | `apps.systemUser.credsSecretName` | Name of a pre-created secret containing credentials of the system user (see [Handling Secrets](/deployment/secrets) topic for expected secret content) | `SYSTEM` |
+| `apps.tuningDefaults.coreThreads` | Minimum number of threads to associate with the [executor](https://www.ibm.com/support/knowledgecenter/SSEQTP_liberty/com.ibm.websphere.liberty.autogen.base.doc/ae/rwlp_config_executor.html) | `4` |
+| `apps.tuningDefaults.maxThreads` | Maximum number of threads that can be associated with the [executor](https://www.ibm.com/support/knowledgecenter/SSEQTP_liberty/com.ibm.websphere.liberty.autogen.base.doc/ae/rwlp_config_executor.html) | `4` |
+| `apps.tuningDefaults.maxPoolSize` | Maximum number of physical connections for a [pool](https://www.ibm.com/support/knowledgecenter/SSEQTP_liberty/com.ibm.websphere.liberty.autogen.base.doc/ae/rwlp_config_connectionManager.html) | `8` |
+| `apps.tuningDefaults.numConnectionsPerThreadLocal` | Number of connections to be cached for each thread | `2` |
```
Comment (ourboy) [inline src/pages/deployment/config-reference.mdx:85]: missing link to external documentation. should ling to https://www.ibm.com/docs/en/was-liberty/base?topic=configuration-connectionmanager

**Review by ourboy — CHANGES_REQUESTED** [review]

**Review by gillesde — APPROVED** [review]

**Review by ourboy — APPROVED** [review]

**Pull request merged**